zebra-consensus/src/checkpoint.rs

```diff
@@ -35,7 +45,41 @@ struct CheckpointVerifier<S> {
     /// which only happen in the last few hundred blocks in the chain.
     /// (zcashd allows chain reorganizations up to 99 blocks, and prunes
     /// orphaned side-chains after 288 blocks.)
+    ///
+    /// Checkpoints must start at zero, and have equal spacing. There must
+    /// be at least one checkpoint, the genesis block.
     checkpoint_list: Arc<HashMap<BlockHeight, BlockHeaderHash>>,
+
+    // Derived Variables
+    //
+    /// The height of the maximum checkpoint.
+    ///
+    /// Can be zero, if there is a single genesis block checkpoint.
+    max_checkpoint_height: BlockHeight,
+
+    /// The fixed spacing between checkpoints.
+    ///
+    /// Can be zero, if there is a single genesis block checkpoint.
+    checkpoint_frequency: BlockHeight,
```

Comment:
nit: I think this should probably be a `usize`, since it's not actually a height but a frequency.

Reply:
I think we can eliminate this field entirely:
https://github.com/ZcashFoundation/zebra/pull/524#discussion_r443934552

Reply:
Removed in 293e9d17c5672782e2b0adf50b74979fae417a14.